### DISCUSSION TITLE
Row selection checkbox not highlighted V7

### DISCUSSION THREAD

**Discussion by chrisbekas** [icflorescu/mantine-datatable#463] [Q&A]
Clicking a row selection checkbox in V7 does not highlight the checkbox background like it did in V6 (which I quite liked!). 

Is there a way to have that behave like it did in V6? Thank you!

**Comment by icflorescu** [comment]
Thanks for the question, @chrisbekas!

I am aware of the issue, but didn't get a chance to address it yet, because it's not as straightforward to implement as it used to be in V6 because of the new Mantine styling internals.

Here's why:

To respect the Mantine theme, I'm using `var(--mantine-primary-color-light)` as selection highlight color on cells (`<td/>`s), which is a transparent shade (of blue in the default theme).

However, I have to override the selection cell (`<td />`) background and set it to `inherit` the parent row color, because if I don't, I'd end with something like this when the table is horizontally scrolled:

![image](https://github.com/icflorescu/mantine-datatable/assets/581999/2cc7b9fb-8aa5-4508-8fd6-9f53d6354913)

Which is obviously not ok.

Now that I think about it the same issue affects the last column when using the new `pinLastColumn` feature that I added yesterday.
There might be a way to fix this by adding a pseudo-element on top of it, but behind the checkbox.
So, I'll turn this into an issue and I'll see what I can do about it.

**Comment by icflorescu** [comment]
This is fixed in `v7.1.5` 🎉

![image](https://github.com/icflorescu/mantine-datatable/assets/581999/5bb3aa06-2acb-4e04-990d-62e0b98cf6f2)
